### ISSUE TITLE
RIAssigner from comments: add pre-cleaning function

### ISSUE THREAD

**Issue by hechth** [RECETOX/galaxytools#495]
We should add an option that is turned on by default to the tool to ask whether existing RI information should be cleaned beforehand to avoid situations where an old RI is still present in spectra that don't have anything listed in the comments after the new one has been assigned to other spectra.

This should be done using the RIAssigner tool to just set the RI to non prior to assigning it from the comments. This should be covered by a test case.

Questions:
- should this be optional or always done?
  - benefits: user can decide whether they want to do that or not, no data is removed
  - disadvantages: confusing and encouraging inconsistent behaviour
- Should the key be completely removed or set to None?

Example spectrum where this is the case:

```
FORMULA: C2H6
COMMENT: Any=200 ; NIST MS# 23, Seq# M168 |RI:220|
INCHI: InChI=1S/C2H6/c1-2/h1-2H3
SMILES: CC
IONMODE: positive
COMPOUND_NAME: Ethane
RETENTION_INDEX: 220.0
RETENTION_TIME: None
ADDUCT: [M]+
PRECURSORMZ: 29.99945142009076
NUM PEAKS: 14
2.0         2.0
12.0        4.0
13.0        9.99
14.0        29.97
15.0        43.96
16.0        1.0
24.0        5.0
25.0        34.97
26.0        231.79
27.0        331.7
28.0        999.0
29.0        214.81
30.0        261.76
31.0        5.0

```

Advantages for implementing it in the package
- consistency between galaxy tool and package
- easier to use

Advantages for implementing it in the Galaxy tool:
- avoid release cycle for package

So basically it makes sense to implement this first in the package and then add it here in the galaxy wrapper.
